src/term.rs

```diff
@@ -188,14 +191,16 @@ pub struct Contract {
 }
 
 #[derive(Debug, PartialEq, Clone)]
-pub struct MetaValue {
+pub struct AbsMetaValue<V> {
```

Comment:
I haven't seen `AbsMetaValue` used anywhere, in the end. Did you end up not needing it?

Reply:
Oups, finaly I didn't.

Reply:
I mean, now that it's done it doesn't hurt to keep it, but I was just wondering.

Reply:
I finaly droped it. Because I was not realy convinced if the `Option` should be part of the `AbsMetaVlue` or as Type parameter. Better if we make it when we realy will need it. It's 2 lines to update and will not break the previous code.